app/src/main/res/layout/fragment_body.xml

```diff
@@ -40,7 +40,8 @@
                     <android.support.v7.widget.CardView
```

Comment:
Why is there a CardView here in the first place?

Reply:
Good point ... lets keep this open till this is refactored

Reply:
It does give a shadow to the button bar underneath, so it might be good to keep it.